design/overview.md

```diff
@@ -73,4 +78,14 @@ Note: X% is some security factor for bootstrapping. 67% is sufficient to release
 
 ## Relaying Cosmos to ETH
 
-**TODO**
\ No newline at end of file
+* Someone (permissionlessly) sends MsgSendToEth this creates an entry in the 'peggy bridge tx pool' this pool is in the Cosmos chain store and contains all transactions not yet bridged. Funds of the specified demon are then burned from the users account if that demon is valid to bridge, otherwise the tx is rejected. 
+* Someone (permissionlessly) sends a MsgRequestBatch, this produces a new `Transaction batch` in the `Peggy Bridge Tx pool`. The creation of this batch occurs in ComsosSDK and is entierly deterministic, ordering transactions from highest to lowest fee and including them in the `Transaction batch` until there are no more transactions or the batch size limit is reached.
+* The `Orchestrator Set` observes the `Peggy Bridge Tx pool` via REST endpoints. When a new batch appears they sign the batch and submit a MsgConfirmBatch containing their Ethereum signature over the batch.
+* Once enough signatures are collected (66% of voting power) any `Orcestrator` can collect these signatures from the REST endpoint, assemble an Ethereum transaction and submit the batch.  
```

Comment:
There are a few steps of details missing here. The SDK side is specified, but the Ethereum side is not. Everything here is correct. I would add a few more points here (in this item or as separate items).

* The orchestrator submitting the Ethereum transaction must pay the gas price in ETH and collects the total of all fees in ERC20 tokens. This leads to some detailed handling of exchange rates (see "questions" section)
* The Ethereum transaction requires signatures of 67% of the last registered **MultiSig Set**. This is usually close to the **Orchestrator Set**, but may deviate a lot of some big stake shifts happened. The last MultiSig Set should be stored in Cosmos side as well to validate if we have received enough votes. All validators in the last **MultiSig Set** are allowed to submit `MsgConfirmBatch`, even if they are no longer in the **Orchestrator Set**